template.tex

```diff
@@ -54,20 +54,27 @@
 % Social icons
 \social[linkedin]{john.doe}                        % optional, remove / comment the line if not wanted
 \social[xing]{john\_doe}                           % optional, remove / comment the line if not wanted
-\social[twitter]{ji\_doe}                             % optional, remove / comment the line if not wanted
+
 \social[github]{jdoe}                              % optional, remove / comment the line if not wanted
 \social[gitlab]{jdoe}                              % optional, remove / comment the line if not wanted
-\social[stackoverflow]{0000000/johndoe}            % optional, remove / comment the line if not wanted
+\social[codeberg]{jdoe}                            % optional, remove / comment the line if not wanted
 \social[bitbucket]{jdoe}                           % optional, remove / comment the line if not wanted
+\social[stackoverflow]{0000000/johndoe}            % optional, remove / comment the line if not wanted
+
 \social[skype]{jdoe}                               % optional, remove / comment the line if not wanted
 \social[orcid]{0000-0000-000-000}                  % optional, remove / comment the line if not wanted
 \social[researchgate]{jdoe}                        % optional, remove / comment the line if not wanted
 \social[researcherid]{jdoe}                        % optional, remove / comment the line if not wanted
+\social[googlescholar]{googlescholarid}            % optional, remove / comment the line if not wanted
+
+\social[twitter]{ji\_doe}                          % optional, remove / comment the line if not wanted
+\social[mastodon]{mastodon.social/web/@user}       % optional, remove / comment the line if not wanted
```

Comment:
I guess the reason why we don't add the full link here is because that mastodon is federated?

Reply:
Yes - mastodon.social is just one of many possibilities (e.g. I am at fosstodon.org)